src/backend/distributed/sql/udfs/citus_split_shard_by_split_points/latest.sql

```diff
@@ -0,0 +1,22 @@
+DROP TYPE IF EXISTS citus.split_mode;
+
+-- Three modes to be implemented: blocking, non_blocking and auto.
+-- Currently, the default / only supported mode is blocking.
+CREATE TYPE citus.split_mode AS ENUM (
```

Comment:
Do we want a new type for this? We already have citus.shard_transfer_mode which seems very similar.